tests/benchmark.py

```diff
@@ -116,6 +116,31 @@ def add_xyz_rxns_from_file(filename):
 
 
 """
+===============================================================================
```

Comment:
is this generated automatically?

Reply:
Not really
```
python tests/benchmark.py -sm && python tests/benchmark.py -so 
```
would be nice if it was but needs non CI compute. Or have I misunderstood what you meant

Reply:
I just mean is the text output automatically generated when you run that^

Reply:
ah, yes 👍